src/NodaTime/TimeZones/IDateTimeZoneSource.cs

```diff
@@ -100,6 +100,7 @@ public interface IDateTimeZoneSource
         /// The ID for the given system time zone for this source, or null if the system time
         /// zone has no mapping in this source.
         /// </returns>
+        [Obsolete("Only the system default time zone can be mapped in 2.0, using GetSystemDefaultId. For other time zones, use source-specific members.")]
```

Comment:
Well, in 2.0 it's not an extension method - and the first half of the sentence implies that it's talking about 2.0. I'd prefer to leave it slightly woolly to cover both 1.4 and 2.0.